Lib/pathlib.py

```diff
@@ -1127,6 +1127,12 @@ def lchmod(self, mode):
         """
         self.chmod(mode, follow_symlinks=False)
 
+    def chown(self, uid, gid, *, follow_symlinks=True):
+        """
+        Change the owner and group id of path to the numeric uid and gid, like os.chown().
+        """
```

Comment:
Please follow PEP 8.

```suggestion
        """Change the owner and group id of path to the numeric uid and gid, like os.chown()."""
```